.github/workflows/release.yml

```diff
@@ -12,6 +12,21 @@ on:
         description: 'HomeBrew prerelease'
         default: 'false'
 jobs:
+  notify-start:
+    name: Notify Release Started
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Slack Notification
+        uses: rtCamp/action-slack-notify@v2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEB_HOOK }}
+          SLACK_COLOR: "#36a64f"
+          SLACK_USERNAME: CLI Release Bot
+          SLACK_ICON_EMOJI: ":ship:"
+          SLACK_TITLE: "Twilio Cli"
```

Comment:
I see the SLACK_TITLE for ` Notify Release Failed` being `Twilio CLI-Core` shouldn't this be the same or specific to twilio cli core?